src/biogeophys/SoilHydrologyMod.F90

```diff
@@ -425,7 +425,7 @@ subroutine Infiltration(bounds, num_hydrologyc, filter_hydrologyc, num_urbanc, f
 
              !7. remove drainage from h2osfc and add to qflx_infl
              h2osfc(c) = h2osfc(c) - qflx_h2osfc_drain(c) * dtime
-             qflx_infl(c) = qflx_infl(c) + qflx_h2osfc_drain(c)
+             qflx_infl(c) = qflx_infl(c) + frac_h2osfc(c)*qflx_h2osfc_drain(c)
```

Comment:
**Comment from @martynpclark Aug 17, 2017**

Sub-grid fractions are required when calculating the area=average infiltration.